ports/servoshell/minibrowser.rs

```diff
@@ -241,6 +251,19 @@ impl Minibrowser {
                         })),
                     });
                 });
+            // Show status tooltip if there is status text
+            if let Some(status_text) = status_text_clone.as_ref() {
+                let screen_rect = ctx.available_rect();
```

Comment:
I think ctx is not defined within this scope. I would suggest moving this block of code to a section where ctx is defined, such as after line 156.